### PR TITLE
fix: Update theme to minimum version notice issue.

### PR DESCRIPTION
**Pull request by premanshup**

### CODE REVIEW

File: inc/theme-update/class-astra-theme-background-updater.php [hunk 1]
```diff
@@ -115,6 +115,18 @@ private function needs_db_update() {
...
+		/**
+		 * Is the DB updated?
+		 *
+		 * @since x.x.x
+		 * @return boolean
+		 */
+		private function is_db_updated() {
+			$current_theme_version = astra_get_option( 'theme-auto-version', null );
+
+			return ! is_null( $current_theme_version ) && version_compare( $current_theme_version, ASTRA_THEME_VERSION, '=' );
```
Comment (Balachandark) [inline inc/theme-update/class-astra-theme-background-updater.php]: `! is_null( $current_theme_version )` condition is can be skipped as we are already using `is_new_install` function for it.

File: inc/theme-update/class-astra-theme-background-updater.php [hunk 2]
```diff
@@ -132,20 +144,23 @@ private function update() {
 			$current_db_version = astra_get_option( 'theme-auto-version' );
 
 			error_log( 'Batch Process Started!' );
-			foreach ( $this->get_db_update_callbacks() as $version => $update_callbacks ) {
-				if ( version_compare( $current_db_version, $version, '<' ) ) {
-					foreach ( $update_callbacks as $update_callback ) {
-						error_log( sprintf( 'Queuing %s - %s', $version, $update_callback ) );
-
-						self::$background_updater->push_to_queue( $update_callback );
+			if ( count( $this->get_db_update_callbacks() ) !== 0 ) {
```
Comment (patilvikasj) [inline inc/theme-update/class-astra-theme-background-updater.php]: The better condition will be `$this->get_db_update_callbacks() ) > 0 `

File: inc/theme-update/class-astra-theme-background-updater.php (hunk 1 above)
Comment (premanshup) [reply inc/theme-update/class-astra-theme-background-updater.php]: Done!

File: inc/theme-update/class-astra-theme-background-updater.php (hunk 2 above)
Comment (premanshup) [reply inc/theme-update/class-astra-theme-background-updater.php]: Done!

File: inc/theme-update/class-astra-theme-background-updater.php [hunk 3]
```diff
@@ -71,10 +71,12 @@ public function install_actions() {
 
 			$is_queue_running = astra_get_option( 'is_theme_queue_running', false );
 
-			if ( $this->needs_db_update() && ! $is_queue_running ) {
+			if ( $this->needs_db_update() && ! $is_queue_running || ! $this->is_db_updated() ) {
```
Comment (patilvikasj) [inline inc/theme-update/class-astra-theme-background-updater.php]: Needs $this->needs_db_update() and ! $is_queue_running condition in parenthesis .

**Pull request merged**